src/CCPACSPointListXYZ.cpp

```diff
@@ -68,4 +71,29 @@ std::vector<CTiglPoint>& CCPACSPointListXYZ::AsVector()
     return m_vec;
 }
 
+void CCPACSPointListXYZ::OrderPoints()
+{
```

Comment:
@bernhardmgruber From my point of view, this does not belong here!

We cannot assume any order of points without knowing, what the points are used for. In case of a wing, this is known (we can make assumptions). Generally, this is not the case.

Please revert it, as it also does not seem to be related to the caching stuff.

Reply:
@rainman110 I know, and I remember moving this method elsewhere. Strangly, I cannot find the change in later commits so I guess I only applied it internally and not moved it back to your TiGL. I'll try to come up with something.

Reply:
I moved OrderPoints back into CCPACSWingProfile. I guess this should be fine then.

Reply:
@bernhardmgruber The first job succeeded. Looks good :smile: